projects/packages/my-jetpack/src/products/class-backup.php

```diff
@@ -209,10 +209,12 @@ public static function get_post_activation_url() {
 	 * @return ?string
 	 */
 	public static function get_manage_url() {
-		if ( static::is_jetpack_plugin_active() ) {
-			return Redirect::get_url( 'my-jetpack-manage-backup' );
-		} elseif ( static::is_plugin_active() ) {
+		// check standalone first
+		if ( static::is_standalone_plugin_active() ) {
 			return admin_url( 'admin.php?page=jetpack-backup' );
+			// otherwise, check for the main Jetpack plugin
+		} elseif ( static::is_jetpack_plugin_active() ) {
+			return Redirect::get_url( 'my-jetpack-manage-backup' );
```

Comment:
I think that, for the Boost issue I mentioned above, it fell under this condition (for some reason, skipping the check above) and ended up redirecting to the Backup page.

Reply:
When you got sent to the backup page, was it from this screen?
![Screenshot 2024-02-27 at 2 55 13 PM](https://github.com/Automattic/jetpack/assets/18016357/d52ede47-2125-488c-83cc-88c4e9d14031)

If so, I think it was a separate issue that I am looking at here: https://github.com/Automattic/wp-calypso/pull/87961